example/main.go

```diff
@@ -27,6 +29,10 @@ func main() {
 	fmt.Println(flag.Args())
 
 	p := js.Global().Get("document").Call("createElement", "p")
-	p.Set("innerText", "Hello, World!")
+	p.Set("innerText", strings.Join([]string{
+		"Hello, World!",
+		fmt.Sprintf("args=%q", os.Args),
+		fmt.Sprintf("env=%q", os.Environ()),
```

Comment:
`Args` and `Environ()` are `[]string` so `%q` would not work (IIUC). Maybe `%v` is better?

Reply:
`%q` prints them like `["./example"]`, while `%v` prints like `[./example]`. I like having the quotes because it helps distinguish arguments or variables with spaces in them.

Reply:
I see, thanks!